gangsta_chess/logic/board/chess_board.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+from gangsta_chess.logic.piece.piece import Piece
+from gangsta_chess.logic.board.board import Board
+from gangsta_chess.logic.piece.pawn import Pawn
+from gangsta_chess.logic.piece.bishop import Bishop
+from gangsta_chess.logic.piece.king import King
+from gangsta_chess.logic.piece.knight import Knight
+from gangsta_chess.logic.piece.queen import Queen
+from gangsta_chess.logic.piece.rook import Rook
+
+
+class ChessBoard(Board):
+    def __init__(self, Pieces, shape=(8, 8)):
+        super().__init__(Pieces, shape)
+        self.back_row = np.array([Rook, Knight, Bishop, King, Queen, Bishop, Knight, Rook])
+
+    def place_pieces(self):
+        self.board[0] = np.array([self.back_row[i]((0, i), 'w', self) for i in range(8)])
+        self.board[1] = np.array([Pawn((1, i), 'w', self) for i in range(8)])
```

Comment:
Maybe add a front row instance variable to the constructor then add a front row argument with a default value to support dependency injection. It's a bit hidden in here.